Components/BlockImageTextParallax/script.js

```diff
@@ -0,0 +1,51 @@
+import Rellax from 'rellax'
+import $ from 'jquery'
+
+const $window = $(window)
+
+let rellax, initialized
+
+class BlockImageTextParallax extends window.HTMLDivElement {
+  constructor (self) {
+    self = super(self)
+    self.$ = $(self)
+    return self
+  }
+
+  connectedCallback () {
+    initParallax()
+  }
+}
+
+function initParallax () {
+  if (!initialized) {
+    initialized = true
+    setParallax()
+    $window.on('resize', setParallax)
+  }
+}
+
+function setParallax () {
+  if (isMobile()) {
+    if (rellax) {
+      rellax.destroy()
+      rellax = false
+    }
+  } else {
+    if (!rellax && !isMobile()) {
+      rellax = new Rellax('[data-parallax]', {
```

Comment:
Please consider the scope of this selector. Other components using the same data attribute would not work as expected like this. Also if you wanted to add custom options for a single instance of this component, this could become a problem.

Reply:
_ignored this one as agreed_